fulp_modules/features/sect/bull.dm

```diff
@@ -0,0 +1,50 @@
+/mob/living/basic/bull
```

Comment:
Should this be a subtype of cows? They seem to share some things in common that you can make use of